funcx_web_service/routes/funcx.py

```diff
@@ -655,10 +640,10 @@ def get_ep_stats(user: User, endpoint_id):
 
     try:
         if not authorize_endpoint(user_id, endpoint_id, None, token):
-            return create_error_response(EndpointAccessForbidden(endpoint_id), jsonify_response=True)
+            raise EndpointAccessForbidden(endpoint_id)
     except Exception as e:
         # could be EndpointNotFound
-        return create_error_response(e, jsonify_response=True)
+        raise e
```

Comment:
Is there value in catching and immediately re-raising?

Reply:
Nope, removed this.